src/dbtPowerUserExtension.ts

```diff
@@ -82,8 +82,8 @@ export class DBTPowerUserExtension implements Disposable {
 
   async activate(context: ExtensionContext): Promise<void> {
     this.dbtProjectContainer.setContext(context);
-    this.dbtProjectContainer.initializeWalkthrough();
```

Comment:
Should be fixed so it works dynamically, ie when projects are being added/removed